samples/kvsWebRTCClientMaster.c

```diff
@@ -216,12 +218,18 @@ PVOID sendVideoPackets(PVOID args)
             goto CleanUp;
         }
 
+        // based on bitrate of samples/h264SampleFrames/frame-*
+        encoderStats.width = 640;
+        encoderStats.height = 480;
+        encoderStats.targetBitrate = 262000;
         frame.presentationTs += SAMPLE_VIDEO_FRAME_DURATION;
 
         if (!ATOMIC_LOAD_BOOL(&pSampleConfiguration->updatingSampleStreamingSessionList)) {
             ATOMIC_INCREMENT(&pSampleConfiguration->streamingSessionListReadingThreadCount);
             for (i = 0; i < pSampleConfiguration->streamingSessionCount; ++i) {
                 status = writeFrame(pSampleConfiguration->sampleStreamingSessionList[i]->pVideoRtcRtpTransceiver, &frame);
+                encoderStats.encodeTimeMsec = 4; // update encode time to an arbitrary number to demonstrate stats update
```

Comment:
Perhaps 40 ms is a better choice here due to 25 fps?

Reply:
its not "duration" its how much time encoder spent encoding the frame 
setting it to 40ms will be misleading

Reply:
Oh, ok. Makes sense